src/server_manager/web_app/digitalocean_app/model/account.ts

```diff
@@ -0,0 +1,212 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {EventEmitter} from 'eventemitter3';
+
+import * as crypto from '../../../infrastructure/crypto';
+import * as do_install_script from '../../../install_scripts/do_install_script';
+import * as account from '../../../model/account';
+import {Account, DigitalOceanCredentials, DigitalOceanLocation, DigitalOceanStatus} from '../../../model/account';
+import {CloudProviderId} from '../../../model/cloud';
+import {ManagedServer} from '../../../model/server';
+import {ShadowboxSettings} from '../../shadowbox_server';
+import {DigitalOceanApiClient, HttpError, NetworkError} from '../infrastructure/api';
+
+import {DigitalOceanServer} from './server';
+
+const SHADOWBOX_TAG = 'shadowbox';
+const MACHINE_SIZE = 's-1vcpu-1gb';
+
+// TODO: Cache account data so that we don't fetch on every request.
+export class DigitalOceanAccount implements account.DigitalOceanAccount {
```

Comment:
This file is not really a model, since it is providing an implementation. Maybe move to `digitalocean_app/account.ts`? (Same comment for all files in this directory).